packages/node-cli/messages.json

```diff
@@ -282,6 +282,10 @@
 	"UTILS_ACCOUNT_SPECIFIC_VALUES_ARGUMENT_HANDLER_ERRORS_WRONG_ACCOUNT_SPECIFIC_VALUES_OPTION": "You have specified an invalid value for the \"--accountspecficvalues\" option. Enter either WARNING or ERROR.",
 	"UTILS_APPLY_CONTENT_PROTECTION_ARGUMENT_HANDLER_ERRORS_APPLY_CONTENT_PROTECTION_IN_ACP": "Content protection cannot be applied to account customization projects.",
 	"UTILS_APPLY_CONTENT_PROTECTION_ARGUMENT_HANDLER_ERRORS_APPLY_CONTENT_PROTECTION_WITHOUT_HIDING_AND_LOCKING": "There was an error with the {0} command. Content protection was not applied because the hiding.xml and locking.xml files were not found in the \"InstallationPreferences\" folder of the project. You need to create both files in order to apply content protection.\nTo learn how to create a hiding.xml file, see {1}.\nTo learn how to create a locking.xml file, see {2}.",
+	"UTILS_APPLY_CONTENT_PROTECTION_ARGUMENT_HANDLER_ERRORS_APPLY_CONTENT_PROTECTION_WITH_APPLY_INSTALLATION_PREFERENCES": "The \"--applycontentprotection\" and \"--applyinstallprefs\" options are mutually exclusive. Use one of the two at a time.",
+	"UTILS_APPLY_CONTENT_PROTECTION_IS_DEPRECATED": "The \"--applycontentprotection\" option will be removed in NetSuite 2021.2. You should use the \"--applyinstallprefs\" option starting from NetSuite 2021.1.",
+	"UTILS_APPLY_INSTALLATION_PREFERENCES_ARGUMENT_HANDLER_ERRORS_APPLY_INSTALLATION_PREFERENCES_IN_ACP": "Installation preferences cannot be applied to account customization projects.",
+	"UTILS_APPLY_INSTALLATION_PREFERENCES_ARGUMENT_HANDLER_ERRORS_APPLY_INSTALLATION_PREFERENCES_WITHOUT_HIDING_LOCKING_AND_OVERWRITING": "There was an error with the {0} command. Installation preferences were not applied because the hiding.xml, locking.xml, and overwriting.xml files were not found in the \\\"InstallationPreferences\\\" folder of the project. You need to create the three files to apply installation preferences.\\nFor more information about installation preferences, see {1}",
```

Comment:
I see that this is a wrapping of another message (COMMAND_DEPLOY_MESSAGES_NOT_ASKING_INSTALLATION_PREFERENCES_REASO) . Could we use that one and do the wrapping outside in the code?

Reply:
The problem is that COMMAND_DEPLOY_MESSAGES_NOT_ASKING_INSTALLATION_PREFERENCES_REASON has an embedded message itself, so I don't think it would work unless we structure the messages differently

Reply:
Well, I think it would work okay. If not, you could always concatenate strings.
Anyways it's not a big deal. Feel free to change it or just move it to ready for testing